### ISSUE TITLE
Chatter.is_subscriber returns '0', type 'str' instead of boolean.

### ISSUE THREAD

**Issue by aticie** [TwitchIO/TwitchIO#213]
`Chatter.is_subscriber` property states that it returns boolean in docs:
```python
    @property
    def is_subscriber(self) -> Optional[bool]:
        """A boolean indicating whether the User is a subscriber of the current channel.

        Could be None if no Tags were received.
        """
        return self._sub
```

But currently, `message.author.is_subscriber`, it is '0' for non subscribers. This causes issues even when you try to cast it to boolean:

```python
>>> bool(message.author.is_subscriber)
True
```
# Expected behavior

`.is_subscriber` should return boolean False when the Chatter is not a subscriber.

**Comment by chillymosh** [comment]
This will be fixed in next release of 2.1.0

**Comment by IAmTomahawkx** [comment]
Fixed in 98a89f1
